test/support/content_register_helpers.rb

```diff
@@ -27,6 +41,13 @@ def policy_1
         "format" => "policy",
         "title" => "Policy 1",
         "base_path" => "/government/policies/policy-1",
+        "links" => {
+          "related" => [
```

Comment:
Ah yeah, you're right. I was under the assumption that they were in the related field.